module/organisations/validator/v1.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (req) => {
+
+    let organisationValidator = {
+
+        addUser: function () {
+            req.checkBody('userId').exists().withMessage("required userId");
+            req.checkBody('roles').exists().withMessage("required roles");
+            req.checkBody('lastName').exists().withMessage("required organisation"); 
```

Comment:
@rakeshSgr - Check body has lastname and message has organisation

Reply:
resolved